src/libcore/ptr.rs

```diff
@@ -74,7 +74,10 @@ pub const fn null_mut<T>() -> *mut T { 0 as *mut T }
 ///
 /// # Safety
 ///
-/// This is only unsafe because it accepts a raw pointer.
+/// This function copies the memory through the raw pointers passed to it
+/// as arguments, which means they will be dereferenced.
```

Comment:
I don't think the warning is clear enough. I'd state something like:

"If the pointer is null or points to invalid data, the program will crash or have an undefined behaviour."

It seems very important to me to indicate the consequences.